lighthouse-cli/cli-flags.js

```diff
@@ -152,6 +161,22 @@ function getFlags(manualArgv) {
       .default('enable-error-reporting', undefined) // Undefined so prompted by default
       .default('channel', 'cli')
       .check(/** @param {LH.CliFlags} argv */ (argv) => {
+        // ".middleware" does not exist in this version of yargs, so do some preprocessing here.
```

Comment:
still not 100% sure that I love mutating the arguments in a `.check` method 

WDYT of just saving the `.argv` to a variable and mutating *that* before we return in `getFlags`?

I think that might help make it more explicit that we are doing some non-standard yargs mutations

Reply:
done